### PR TITLE
Chrome options V2

### PR DESCRIPTION
**Pull request by PaulCombal**
I completed the options V2 thing, it doesn't look very good, it's very basic, and the code isn't the finest, but it's functionnal. I'm running out of time this weekend, so it's up to you to modify what you want on the chromeOptions branch, if you want to add/modify features.

Here is what it looks like for now:
image

Please don't merge now, as the "append to document" part is not finished, or merge into a development branch.
I can't really continue working on the extension, because I just got a 24 hours ban for suspicious traffic :/ (I guess I refreshed too much, and the alarm triggered too many times as I reload the extension)
Here are my attempts at trying to get settings.html's body with jquery:

`$.get(chrome.extension.getURL('/settings.html'), function(settingsDiv){

		var settdiv = $.parseHTML(settingsDiv);

		console.log($(settingsDiv).filter("#settingsDiv").html());

		console.log($(settingsDiv).filter($("body")).html());

		settdiv = $(settdiv).find("#settingsDiv");`

If I had more time, I guess I could move the fillSettingsDiv function to another file or something that could be used both by chrome options and on page options.
To sum up:
Working Options V2 - Done
Still works with on page options - Todo

Good luck

### CODE REVIEW

File: backgroundpage.js [hunk 1]
```diff
@@ -239,7 +239,7 @@ var arr = [],
 
 /*Creating a new tab if notification is clicked*/
 chrome.alarms.create("routine", {
-    delayInMinutes: .1
+    delayInMinutes: 1
```
Comment (PaulCombal) [inline backgroundpage.js]: It keeps telling me the minimum is 1, and that in the packed extension it will be set to 1.

File: chromeOptions.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+			repeatHours: $('#hoursField').val(),
+			RepeatHoursBG: parseInt($('#hoursFieldBG').val()), //parseInt to save 0.5 as 0
+			Pagestoload: $('#pagestoload').val(),
+			PagestoloadBG: $('#pagestoloadBG').val(),
+			PageForBG: $('#pageforBG').val(),
+			DelayBG: $('#delayBG').val(),
+			MinLevelBG: $('#minLevelBG').val(),
+			ShowChance: $('#chkShowChance').is(':checked').toString()
+		}, function(){
+			alert("Settings Saved!")
```
Comment (PaulCombal) [inline chromeOptions.js]: This is either annoying or doesnt work. That should be changed

**Comment by ge-ku** [comment]
Eh, it's fine, I will not upload it into Chrome store until it works. Good job.
